pontoon/base/models.py

```diff
@@ -99,7 +99,7 @@ def user_profile_url(self):
 
 
 def user_gravatar_url(self, size):
-    email = hashlib.md5(self.email.lower()).hexdigest()
+    email = hashlib.md5(self.email.lower().encode('utf-8')).hexdigest()
```

Comment:
Will we be able to remove those `.encode('utf-8')` once we're on Py3?

Reply:
Py3 reduces it to `.encode()`. Hashlib requires bytes here.